src/components/menuItemPropTypes.js

```diff
@@ -10,6 +10,9 @@ const propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     wrapperStyle: stylePropTypes,
 
+    /** Used to apply offline styles to child text components */
+    style: stylePropTypes,
```

Comment:
Oh! Did not know we had this, can you please use it in OfflineWithFeedback.js in lines 42 and 48?